com.unity.render-pipelines.high-definition/Documentation~/Motion-Vectors.md

```diff
@@ -4,33 +4,37 @@ Motion vectors capture the per-pixel, screen-space motion of GameObjects from on
+To use motion vectors in HDRP, you must first enable them:
 
-HDRP calculates motion vectors in two stages:
+1. In your Unity Project’s [HDRP Asset](HDRP-Asset.md):
+   1. Select the HDRP Asset and view it in the Inspector window.
+   2. Go to the **Rendering** section and enable **Motion Vectors**.
+2. In [Frame Settings](Frame-Settings.md). You can either enable motion vectors for all Cameras or on an individual, per-Camera level.
+   1. To enable motion vectors for all Cameras, open the [HDRP Default Settings](Default-Settings-Window.md) Project Settings tab (menu: **Edit** > **Project Settings** > **HDRP Default Settings**), then set **Default Frame Settings For** to **Camera**. To enable motion vectors for a particular Camera, select the Camera and, in the Inspector, enable **Custom Frame Settings**.
+   3. In the **Rendering Section**, enable **Motion Vectors**. This enables camera motion vectors.
+   3. To enable motion vectors for opaque GameObjects, enable **Opaque Object Motion**. To enable motion vectors for transparent GameObjects, enable **Transparent Object Motion**.
 
-1. HDRP first calculates object motion vectors. To make a Mesh Renderer write object motion vectors, in its Inspector, go to **Additional Settings** and select **Per Object Motion** from the **Motion Vectors** drop-down. HDRP now calculates object motion vectors for this Mesh Renderer, if you enable **Object Motion Vectors** in the Frame Settings.
-![](Images/MotionVectors1.png)
-2. HDRP then calculates Camera motion vectors, which are caused by the movement of the Camera. This is for pixels that did not write motion vectors during the first stage. To make HDRP calculate Camera motion vectors for a Mesh Renderer, in the Inspector for the Mesh Renderer, go to **Additional Settings** and select **Camera Motion Only** from the **Motion Vectors** drop-down.
 
-If you do not want HDRP to calculate motion vectors for a Mesh Renderer at all, in the Inspector for the Mesh Renderer, go to **Additional Settings** and select **Force No Motion** from the **Motion Vectors** drop-down.
+HDRP can now render motion vectors. If you enabled object motion vectors, be aware that, by default, new Mesh Renderers write object motion vectors. To change this behavior, select the Mesh Renderer and, in the Inspector, change the value of the **Motion Vectors** property.<br/>![](Images/MotionVectors1.png)
+The options are:
+
+* **Camera Motion Only**: HDRP only calculates camera motion vectors for the area of the screen this GameObject fills.
+* **Per Object Motion**: HDRP calculates object motion vectors for this GameObject.
```

Comment:
This is a bit confusing, the motion vectors that are  computed are the one caused by the object motion (excluding whatever camera motion happens), I don't think it is very clear from this line